service/telemetry/config.go

```diff
@@ -130,6 +130,13 @@ type MetricsConfig struct {
 // TracesConfig exposes the common Telemetry configuration for collector's internal spans.
 // Experimental: *NOTE* this structure is subject to change or removal in the future.
 type TracesConfig struct {
+	// Level determines what level of tracing is enabled across the collector, the
+	// possible values are:
+	//  - "none" indicates that no trace data should be collected;
+	//  - "basic" is the recommended and covers the basics of the service telemetry.
+	//  - "normal" adds some additional spans top of basic.
+	//  - "detailed" adds all available spans to the previous levels.
```

Comment:
Are any of them utilized now?

Also, `configtelemetry.Level` defaults to Basic. Should we have None by default here?